### ISSUE TITLE
Two questions?

### ISSUE THREAD

**Issue by happsky** [jacobgil/pytorch-grad-cam#7]
Thanks for sharing this amazing code. I have two questions, the one is that how to retrain or fine-tune VGG network with my own data, such as faces with different attributes? The other is that how to use the label information?
